app-frontend/src/store/sagas/api/searchedLinks.api.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
```

Comment:
just put all API requests to 
app-frontend/src/api/searchedLinks.api.ts
instead of app-frontend/src/store/sagas/api/searchedLinks.api.ts